cpp/ps0001/cpp_ps0001_soln.cpp

```diff
@@ -0,0 +1,29 @@
+ #include <iostream>
+using namespace std;
+int main()
+{
+ // promptiung user for 3 numbers to add 1/2
+ unsigned short num1;
+ unsigned short num2;
+ unsigned short num3;
+ unsigned short result;
+
+ cout << "please enter the first number \n";
+ cin >> num1;
+ 
+ cout << "please enter the second number \n";
+ cin >> num2; 
+ 
+ cout << "please enter the third number \n";
+ cin >> num3;
+
+ // outputting the result of the summation process 3/3
+ //processing
+ A = x + y + z;
+
+ // output
+ cout << A << "\n";

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [x] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as the program is user-friendly